docs/DeveloperGuide.md

```diff
@@ -563,77 +564,85 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the **System** is the `NurseyBook` the **Actor** is the `User`, and the **Person** is the
 `Nurse` unless specified otherwise)
 
-**Use cases of elderly commands**
+#### Use cases of elderly commands
 
-**UC1: List elderly**
+##### UC1: List elderly
 1. User requests to list elderly.
 2. NurseyBook shows a list of elderly.
 
     Use case ends.
 
 **Extensions**
-* 1a. User requests to find elderly with matching keywords 
-  * 1a1. NurseyBook shows a list of elderly that matches the user's query (by name)
+* 1a. User requests to <u>find elderlies with matching keywords(<a href="#uc5-find-an-elderly">UC5</a>)</u>.
+  * 1a1. NurseyBook shows a list of elderly that matches the user's query (by name).
```

Comment:
If there's a link to another use case, then there is no need for 1a1 and the 'use case ends' line?